src/js/elements/mapchip.js

```diff
@@ -46,14 +47,17 @@ export class MapChip extends Observable {
             return memo;
         }, [])
 
+        groups = groups.filter((g) => {
```

Comment:
Can you move this part and the part above with the reduce into one until method? I need that for the charts as well. 
thanks.

Reply:
done - created a function in utils.js `getFilterGroups`